t/plugin/openfunction/function-example/test-uri/hello.go

```diff
@@ -1,4 +1,6 @@
 /*
+ * Copyright 2022 The OpenFunction Authors.
```

Comment:
No, I mean we should also update the License file for APISIX.

Reply:
Like adding  "hello.go file from OpenFunction/samples: https://github.com/OpenFunction/samples Apache 2.0" in License file ？

Reply:
You can checkout the LICENSE file as an example.